### PR TITLE
Add job to test PyPy venv creation on Windows

### PR DESCRIPTION
**Pull request by silvanocerza**
## Summary

This PR adds a new job to test that PyPy venvs are correctly created on Windows and they contain all the expected binaries.

## Test Plan

Just let CI run.

### CODE REVIEW

**Comment by silvanocerza** [comment]
I didn't expect a stack overflow. 😅

**Comment by zanieb** [comment]
See https://github.com/astral-sh/uv/blob/main/CONTRIBUTING.md#testing-on-windows (which is handled in all of our Windows CI)

**Comment by silvanocerza** [comment]
That was easy, thanks. 🙏 

I should have read the whole `CONTRIBUTING.md`. 😓

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -596,6 +596,71 @@ jobs:
...
+            if not binary_exist(binary):
+              all_found = False
+
+          if not all_found:
+            print("One or more expected executables were not found.")
+            sys.exit(1)
+
+      - name: "Check version"
+        run: |
+          & .venv\Scripts\python --version
```
Comment (zanieb) [inline .github/workflows/ci.yml:655]: Could you add a couple of these that invoke the non- "python" copies of the executable (and for Linux too)

**Review by zanieb — APPROVED** [review]

**Comment by zanieb** [comment]
Nice work.

**Pull request merged**